client/src/components/RatingsReviews/RR.jsx

```diff
@@ -1,14 +1,19 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-//import SumRating from './SumRating.jsx';
+import SumRating from './SumRating.jsx';
+import ReviewsList from './ReviewsList.jsx';
 
 var RR = ({ productId }) => {
   const [meta, setMeta] = useState({});
+  const [display, setDisplay] = useState([]);
+  const [total, setTotal] = useState(0);
+  const [sort, setSort] = useState('relevant');
+
   const getReviewMeta = () => {
     let url = 'http://localhost:3000/reviews/meta';
```

Comment:
You can change this to '/reviews/meta'. You should take out the "localhost:3000" part in case another user wants to use a different port